test/functional/autoupdate/aup-basic.bats

```diff
@@ -7,26 +7,19 @@ load "../testlib"
 
 test_setup() {
 
-	skip "This test is broken on github actions"
-
 	create_test_environment "$TEST_NAME"
 
 }
 
 test_teardown() {
 
-	if [ -z "$TRAVIS" ]; then
-		return
-	fi
-
 	destroy_test_environment "$TEST_NAME"
 
 }
 
-@test "AUT001: Basic test, Check auto-update enable" {
+@test "AUT001: auto-update disable-enable in different root" {
```

Comment:
why did you add "in different root"? because we are using a `--path`?

Reply:
Yes...should I use a different name?

Reply:
all our tests (or almost all) use --path, so I would just not specify that at all.

Reply:
Okay will change that